docs/how_to_guides/how_to_process_previously_recorded_videos.md

```diff
@@ -1,54 +1,82 @@
 
 # How to process pre-recorded synchronized videos with `alpha` GUI
 
-Here's a workflow you can use to pre-process videos recorded with the `pre-alpha (<=v0.0.54)` using the `alpha` 
+This is a workflow you can use to process **any** pre-recorded & synchronized videos.
+
+!!! take-note "Note"
+
+    This software is a work-in-progress, you'll likely have to click around and test things out to get this process to work for the first time. Often, you might click a button and want to go back, but the GUI doesn't have "go-back" capabilities yet :) So in order to go back, simply restart the GUI. You can quickly restart the GUI at any time by selecting the GUI and pressing `ctrl-R`
+
+??? pre-alpha "pre-alpha tips"
+
+    If you've pre-recorded videos using freemocap `pre-alpha (<=v0.0.54)`, these "pre-alpha tips" boxes will contain info to specifically address a "`pre-alpha` -> `alpha GUI`" workflow!
 
 ## Step 1 - Run the GUI  
-Instructions here - https://github.com/freemocap/freemocap#how-to-run-the-alpha-gui 
+Detailed instructions on how to run the GUI using anaconda can be found [on our GitHub](https://github.com/freemocap/freemocap#how-to-run-the-alpha-gui). 
 
-- run  `python src/gui/main/main.py` in a terminal with the proper environment activated
```

Comment:
Maybe a screenshot of the terminal with the environment specified and the command typed and ready to go? 

(you can MS Paint over your `user path` and whatnot for privacy/security

Might also be a good spot for a call out box explaining what to look for (i.e. that the environment is specified, and the terminal is 'in' the freemocap repo